src/components/TodoInfo/TodoInfo.tsx

```diff
@@ -1,4 +1,28 @@
+import classNames from 'classnames';
+import { Todo } from '../../types/Todo';
+import { UserInfo } from '../UserInfo';
+
 // Add the required types and props
-export const TodoInfo = () => (
-  <>TodoInfo markup</>
-);
+type Props = {
+  todo: Todo;
+};
+
+export const TodoInfo: React.FC<Props> = ({ todo }) => {
+  const {
+    user,
+    title,
+    completed,
+  } = todo;
+
+  return (
+    <article className={classNames('TodoInfo',
+      { 'TodoInfo--completed': completed === true })}
+    >
```

Comment:
You can create variables for the class to make your code more readable

Reply:
I think that `completed` is already a boolean type so you don't need to check if is it true or false.
```suggestion
    <article className={classNames('TodoInfo',
      { 'TodoInfo--completed': completed })}
    >
```